src/main/java/uk/gov/justice/digital/hmpps/whereabouts/listeners/SqsDomainEventListener.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.digital.hmpps.whereabouts.listeners
+
+import com.google.gson.Gson
+import io.awspring.cloud.sqs.annotation.SqsListener
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.whereabouts.services.court.VideoLinkBookingService
+
+@Service
+class SqsDomainEventListener(
+  @Qualifier("videoLinkBookingServiceAppScope")
+  private val videoLinkBookingService: VideoLinkBookingService,
+  private val gson: Gson,
+) {
+  companion object {
+    val log: Logger = LoggerFactory.getLogger(this::class.java)
+  }
+
+  @SqsListener("domainevent", factory = "hmppsQueueContainerFactoryProxy")
+  fun handleDomainEvents(requestJson: String?) {
```

Comment:
we normally add a `@WithSpan` here so that we get an event in app insights - see https://github.com/ministryofjustice/hmpps-prisoner-search/blob/bde8f3c3d11237afecd167a90b199ca919ad3c00/hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/listeners/DomainEventListener.kt#L36

Reply:
I will add that. I can assume the value can be any reasonable string like 
'map-devs-hmpps_whereabouts_api_domain_queue' ?